src/GafferDelight/DelightOptions.cpp

```diff
@@ -51,13 +51,15 @@ DelightOptions::DelightOptions( const std::string &name )
 	options->addChild( new Gaffer::NameValuePlug( "dl:bucketorder", new IECore::StringData( "horizontal" ), false, "bucketOrder" ) );
 	options->addChild( new Gaffer::NameValuePlug( "dl:numberofthreads", new IECore::IntData( 0 ), false, "numberOfThreads" ) );
 	options->addChild( new Gaffer::NameValuePlug( "dl:renderatlowpriority", new IECore::BoolData( false ), false, "renderAtLowPriority" ) );
+	options->addChild( new Gaffer::NameValuePlug( "dl:evaluatefile", new IECore::StringData( "" ), false, "evaluateFile" ) );
 
 	// Quality
 
 	options->addChild( new Gaffer::NameValuePlug( "dl:oversampling", new IECore::IntData( 9 ), false, "oversampling" ) );
 	options->addChild( new Gaffer::NameValuePlug( "dl:quality.shadingsamples", new IECore::IntData( 64 ), false, "shadingSamples" ) );
 	options->addChild( new Gaffer::NameValuePlug( "dl:quality.volumesamples", new IECore::IntData( 1 ), false, "volumeSamples" ) );
 	options->addChild( new Gaffer::NameValuePlug( "dl:clampindirect", new IECore::FloatData( 2 ), false, "clampIndirect" ) );
+	options->addChild( new Gaffer::NameValuePlug( "dl:importancesamplefilter", new IECore::IntData( 1 ), false, "importanceSampleFilter" ) );
```

Comment:
Can we use a BoolPlug for this? It seems like it's just an on/off thing as far as the user is concerned.

Implementation-wise, we convert bools to ints automatically in `IECoreDelight::ParameterList` already, so that's one way we could do the conversion.

Reply:
Yes, absolutely! Sorry about that. I was directly translating the NSI spec where importancesamplefilter is listed as int, but treating it as a bool plug makes much more sense. I'll get that fixed asap.

Reply:
That'd be great, thanks!

Reply:
This should be fixed in the latest commit.